.github/workflows/snack-it.yml

```
@@ -0,0 +1,368 @@
+              });
+              console.log(`Assigned issue to @${prAuthor}`);
+            } catch (assignError) {
+              console.log(`⚠️  Could not assign to @${prAuthor}: ${assignError.message}`);
+              console.log('User may not have write access to the repository');
+            }
+            
+            // Get all labels from the PR
+            const prLabels = pr.labels.map(label => label.name);
+            
+            // Add all PR labels to the issue (excluding 'snack-it' which triggered this)
+            const labelsToAdd = prLabels.filter(label => label !== 'snack-it');
+            if (labelsToAdd.length > 0) {
+              await github.rest.issues.addLabels({
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                issue_number: issue.data.number,
+                labels: labelsToAdd
+              });
+              console.log(`Added labels: ${labelsToAdd.join(', ')}`);
+            }
+            
+            // Store issue data for next step and PR comment
+            core.setOutput('issue_number', issue.data.number);
+            core.setOutput('issue_node_id', issue.data.node_id);
+            core.setOutput('pr_number', pr.number);
+            
+            console.log(`✅ Created issue #${issue.data.number}`);
+            return issue.data.number;
+
+      - name: Add issue to project and set status
+        if: steps.check-label.outputs.snack_it == 'true'
+        continue-on-error: true
+        id: add-to-project
+        uses: actions/github-script@v7
+        env:
+          ISSUE_NUMBER: ${{ steps.create-issue.outputs.issue_number }}
+          ISSUE_NODE_ID: ${{ steps.create-issue.outputs.issue_node_id }}
+          PR_NUMBER: ${{ steps.create-issue.outputs.pr_number }}
+        with:
+          github-token: ${{ secrets.GH_PAT_TOKEN_FOR_SNACK_IT }}
+          script: |
+            // Use the issue created in the previous step (avoid race condition)
+            const issueNumber = parseInt(process.env.ISSUE_NUMBER);
+            const issueNodeId = process.env.ISSUE_NODE_ID;
+            const prNumber = parseInt(process.env.PR_NUMBER);
+            
+            console.log(`Working with issue #${issueNumber}`);
+            
+            // Get full issue and PR details for later use
+            const issue = await github.rest.issues.get({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              issue_number: issueNumber
+            });
+            
+            const pr = await github.rest.pulls.get({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              pull_number: prNumber
+            });
+            
+            try {
+              // GraphQL query to add item to project
+              // Project number from URL: https://github.com/orgs/zenml-io/projects/1
+              const projectNumber = 1;
+              const orgName = 'zenml-io';
```

```suggestion 
            try {
              // GraphQL query to add item to project
              // Project number from URL: https://github.com/orgs/zenml-io/projects/1
              const projectNumber = 1;
              const orgName = 'zenml-io';
              
              // Get full issue and PR details for later use
            const issue = await github.rest.issues.get({
              owner: context.repo.owner,
              repo: context.repo.repo,
              issue_number: issueNumber
            });
            
            const pr = await github.rest.pulls.get({
              owner: context.repo.owner,
              repo: context.repo.repo,
              pull_number: prNumber
            });
```

Formatting of this is probably off, but this will prevent silent failures in case it wasn't able to get the issue or PR (the workflow would otherwise just run through without failing in the outer closure).